app/townsquare/management/commands/calculate_activity_scores.py

```diff
@@ -0,0 +1,16 @@
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
```

Comment:
should we schedule this to run every 15 mins in crontab?

Reply:
yup definitely!